### PR TITLE
add simple-step-component

### PR DESCRIPTION
**Pull request by partha6686**
SimpleStepsCreate component shows all the steps on a single page instead of a carousel.

### CODE REVIEW

File: components/features/SimpleStepsCreate.vue [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            <h3 class="step__section-heading">{{ step?.title }}</h3>
+            <p class="step__section-description">
+              {{ step?.description }}
+            </p>
+          </div>
+          <div class="d-flex flex-column flex-lg-row">
+            <nuxt-img
+              :src="step?.imgSrc"
+              class="img-fluid my-auto"
+              alt="Signup with a free account"
```
Comment (dhaladitya108) [inline components/features/SimpleStepsCreate.vue:23]: @partha6686 the alt would be different for different images. You can get this from the props itself

**Review by dhaladitya108 — APPROVED** [review]

File: components/features/SimpleStepsCreate.vue (hunk 1 above)
Comment (partha6686) [reply components/features/SimpleStepsCreate.vue:23]: I think we can add the title as the alt.`:alt="step?.title"`

Comment (dhaladitya108) [reply components/features/SimpleStepsCreate.vue]: we can, but it's not a good practice. Alt is generally the textual representation of an image when the image is not visible.

Comment (partha6686) [reply components/features/SimpleStepsCreate.vue]: yes, got it.

File: components/features/SimpleStepsCreate.vue [hunk 2]
```diff
@@ -0,0 +1,100 @@
+<template>
+  <section class="container py-5 mt-3">
+    <h2 class="section__heading text-center">
+      {{ heading }}
+    </h2>
+
+    <div class="py-5 row">
+      <div v-for="(step,idx) in steps" :key="step?.title" :class="columnClass">
```
Comment (ankitsinghaniyaz) [inline components/features/SimpleStepsCreate.vue:8]: I think the column class doesn't need to be done through javascript. Css is capable enough of handling such things.
cc: @partha6686  @dhaladitya108 

`:class="col-xs-12 col-sm-12 col"`

The `col` class will auto accommodate all the steps equally on larger screen.

File: components/features/SimpleStepsCreate.vue [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    steps: {
+      type: Array,
+      required: true,
+    },
+    stepCount: {
+      type: Number,
+      default: 3,
+      validator(value) {
+        return [2, 3, 4].includes(value)
+      },
```
Comment (ankitsinghaniyaz) [inline components/features/SimpleStepsCreate.vue:49]: nice!

File: components/features/SimpleStepsCreate.vue (hunk 2 above)
Comment (partha6686) [reply components/features/SimpleStepsCreate.vue:8]: Okay, let me check this one.

Comment (partha6686) [reply components/features/SimpleStepsCreate.vue:8]: `class="col-xs-12 col-sm-12 col"`  
Actually, the card is taking the full width. I guess for this to work we have to give some specific width to the cards.

Comment (dhaladitya108) [reply components/features/SimpleStepsCreate.vue:8]: yes, this is done so that we can manually specify the no. of columns in a particular row.

**Pull request merged**